backend/src/controllers/v1/secretsFolderController.ts

```diff
@@ -1,39 +1,62 @@
 import { Request, Response } from "express";
-import { Secret } from "../../models";
-import Folder from "../../models/folder";
-import { BadRequestError } from "../../utils/errors";
+import { Secret, ServiceTokenData } from "../../models";
+import Folder, { TFolderRootSchema } from "../../models/folder";
+import { BadRequestError, UnauthorizedRequestError } from "../../utils/errors";
 import {
   appendFolder,
   deleteFolderById,
   generateFolderId,
   getAllFolderIds,
   getFolderByPath,
   getParentFromFolderId,
-  searchByFolderId,
   searchByFolderIdWithDir,
-  validateFolderName,
+  validateFolderName
 } from "../../services/FolderService";
 import { ADMIN, MEMBER } from "../../variables";
 import { validateMembership } from "../../helpers/membership";
 import { FolderVersion } from "../../ee/models";
 import { EESecretService } from "../../ee/services";
+import { isValidScope } from "../../helpers/secrets";
+import path from "path";
+
+const getFolderWithPathFromId = (folders: TFolderRootSchema, parentFolderId: string) => {
```

Comment:
i see that we are repeating this function which is also defined in `backend/src/controllers/v1/secretImportController.ts`. Let's put this in `backend/src/services/FolderService.ts`?

Reply:
Done